### PR TITLE
Feature/dev 382 update ethminer

### PR DESCRIPTION
**Pull request by thedigitaldesign**
@dsarfati @mkjenkins 
- Not sure if it matters, but when reported to the console Ethminer is hitting stderr and not stdout
- Log sample,
---- <unknown> being reported by ethminer
```
12:30:17.781 > stderr: cu 12:30:17 cuda-0   Generated DAG + Light in 9,832 ms. 7.77 GB 
left.

12:30:17.959 > stderr:  i 12:30:17 <unknown> Job: 1da16c6d... eth-us-west1.nanopool.org [66.42.105.146:9999]

12:30:18.232 > stderr: cu 12:30:18 cuda-1   Generated DAG + Light in 10,283 ms. 4.77 GB left.

12:30:21.158 > stderr:  m 12:30:21 <unknown> 0:00 A0 363.02 Kh - cu0 185.12, cu1 177.90
12:30:26.159 > stderr:  m 12:30:26 <unknown> 0:00 A0 57.75 Mh - cu0 30.50, cu1 27.24
```

### CODE REVIEW

**Comment by dsarfati** [comment]
@thedigitaldesign Right now it doesn't matter which st* stream the logs come in from. If I remember correctly these have always been messed up in ethminer. There are tons of info messages that come on sterror and vice versa.

File: packages/desktop-app/src/Ethminer.ts [hunk 1]
```diff
@@ -50,10 +50,12 @@ export class Ethminer {
     console.log('machineId: ' + id)
 
     let platform = cuda ? '-U' : '-G'
-    this.processName = cuda ? 'ethminer_cuda.exe' : 'ethminer.exe'
+    this.processName = 'ethminer.exe'
+    // this.processName = cuda ? 'ethminer_cuda.exe' : 'ethminer.exe'
```
Comment (dsarfati) [inline packages/desktop-app/src/Ethminer.ts]: Remove commented out line.

Comment (thedigitaldesign) [reply packages/desktop-app/src/Ethminer.ts]: > Are all these `.bin` files required? I don't remember them in our previous versions.

That was everything in the .zip file. I'm guessing they are for different GPU chip sets for AMD:

- Ellesmere (ethash_ellesmere_lws###.bin): AMD's Ellesmere GPU uses the GCN 4.0 architecture and is made using a 14 nm production process at TSMC.
- Tonga (ethash_tonga_lws###.bin)
- GFX900 (ethash_gfx900_lws###.bin): The AMDGPU LLVM back-end is now seeing ‘GFX1010’, and with GFX900 being Vega, GFX1010 could only be Navi.

Comment (thedigitaldesign) [reply packages/desktop-app/src/Ethminer.ts]: > Remove commented out line.
Removed and pushed

**Pull request merged**